### PR TITLE
Service Map Area Fixes

### PR DESCRIPTION
**Pull request by Hunttark**
These fixes are in response to https://github.com/Aurorastation/Aurora.3/pull/15975 and https://forums.aurorastation.org/topic/18084-feedback-thread-for-service-remap/page/2/. As the service map has been merged, I will be providing QOL and general fixes to missed areas/issues.

Thus far,

- Resolved plating missing under bar window
- Resolved bar wood not having the proper lining
- Resolved bar light darkness
- Resolved dining hall darkness
- Resolved hydroponics trash-chute line
- Resolved dance floor alternate colors
- Resolved freezer air sensor
- Resolved freezer scrubber placement
- Resolved missing holopad

I expect to have more reviews/fixes committed by the end of today (3/11/2023), whenever I receive them via the thread.

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

**Comment by Hunttark** [comment]
Beat me to it, thank you

**Review by SleepyGemmy — APPROVED** [review]

File: html/changelogs/Huntime-Service Map Fixes.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - maptweak: "Resolved service area map issues and bugs"
```
Comment (SleepyGemmy) [inline html/changelogs/Huntime-Service Map Fixes.yml:41]:
```suggestion
  - maptweak: "Resolved service area map issues and bugs."
```

File: maps/sccv_horizon/sccv_horizon-2_deck_2.dmm [hunk 2]
```diff
@@ -25689,9 +25704,9 @@
 /turf/simulated/floor/tiled,
 /area/engineering/lobby)
 "mtP" = (
-/obj/machinery/camera/network/service{
-	c_tag = "Dinner - Kitchen Freezer";
-	dir = 4
+/obj/machinery/alarm{
```
Comment (Generalcamo) [inline maps/sccv_horizon/sccv_horizon-2_deck_2.dmm:25707]:
```suggestion
/obj/machinery/alarm/freezer{
```
Otherwise it won't cool the freezer properly.

Comment (SleepyGemmy) [reply maps/sccv_horizon/sccv_horizon-2_deck_2.dmm:25707]: this should be mapped in, for the PR creator's note.

**Review by Alberyk — APPROVED** [review]

**Pull request merged**